src/client/ldBrowser.tsx

```diff
@@ -31,6 +30,7 @@ class LDBrowser extends Component<LDBrowserProps, HocState> {
   }
 
   render() {
+    // eslint-disable-next-line react/react-in-jsx-scope
```

Comment:
Fix for issue 1 - removing `React` on line 1 above fixes the issue, but `eslint` will flag the line below.